core/field_checkbox.js

```diff
@@ -91,6 +91,14 @@ Blockly.FieldCheckbox.prototype.SERIALIZABLE = true;
  */
 Blockly.FieldCheckbox.prototype.CURSOR = 'default';
 
+/**
+ * Used to tell if the field needs to render. Checkbox fields are statically
```

Comment:
"Used to tell if the field needs to be rendered the next time the block is rendered.  Checkbox fields are statically sized, and only need to be rendered at initialization."

^ If this is inaccurate, let's find a more accurate way to say it.

Reply:
Similar/the same wording for other places where you're setting isDirty to false.

Reply:
Fixed